### PR TITLE
Match underscores in GitHub repository names

### PR DESCRIPTION
**Pull request by sgerrand**
:information_desk_person: This change expands the regular expression for repository names to match underscores.

### CODE REVIEW

**Comment by sgerrand** [comment]
@adg: :pencil2:'d the CLA.

File: script.js [hunk 1]
```diff
@@ -33,8 +33,8 @@ limitations under the License.
 	}
 
 	var hashRe = /#.*\/[a-f0-9]+$/;
-	var issueRe = /^https:\/\/github.com\/([A-Za-z0-9-]+\/[A-Za-z0-9-]+\/issues\/[0-9]+)/
-	var pullRe = /^https:\/\/github.com\/([A-Za-z0-9-]+\/[A-Za-z0-9-]+\/pull\/[0-9]+)/
+	var issueRe = /^https:\/\/github.com\/([A-Za-z0-9-]+\/[\w-]+\/issues\/[0-9]+)/;
```
Comment (adg) [inline script.js:36]: Maybe I'm being dense, but how does `\w-` match underscores? Did you mean dashes? Doesn't `\w` match spaces? Is that what we want?

Comment (sgerrand) [inline script.js:36]: > Maybe I'm being dense, but how does \w- match underscores? Did you mean
> dashes? Doesn't \w match spaces? Is that what we want?

You might be thinking of `\s`.

`\w` is a regular expression character class[0] and is the equivalent of
`[A-Za-z0-9_]`. It's shorthand for "word", but a universally recognised one.

[0] https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp#character-classes

**Comment by parkr** [comment]
Would be happy to see this merged in. 👍

**Comment by adg** [comment]
Not sure how I dropped this

**Pull request merged**